tests/unit/amo/components/TestSearch.js

```diff
@@ -416,7 +416,7 @@ describe(__filename, () => {
       expect(mapStateToProps(state)).toEqual({
         context: state.viewContext.context,
         count: state.search.count,
-        filtersUsedForResults: state.search.filters,
+        filtersUsedForResults: {},
```

Comment:
why the change?

Reply:
I undid this change after making a bunch of other changes to `Search`.